.github/workflow/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: Resolwe CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+  schedule:
+    - cron: "30 2 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
```

Comment:
Would it make sense to use `ubuntu-20.04`?

Reply:
Sure. I will test it on the new protocol PR soon to see if it works as expected.